src/index.js

```diff
@@ -72,15 +72,18 @@ class ESLintWebpackPlugin {
     const options = {
       ...this.options,
       exclude: parseFiles(
-        this.options.exclude || 'node_modules',
+        this.options.exclude || [],
         this.getContext(compiler)
       ),
       extensions: arrify(this.options.extensions),
       files: parseFiles(this.options.files || '', this.getContext(compiler)),
     };
 
     const wanted = parseFoldersToGlobs(options.files, options.extensions);
-    const exclude = parseFoldersToGlobs(options.exclude, []);
+    const exclude = parseFoldersToGlobs(
+      this.options.exclude ? options.exclude : '**/node_modules/**',
+      []
+    );
```

Comment:
Just interesting why we have two `exclude`?

Reply:
node_modules cannot be added with the context prefix

Reply:
If the user sets the option to `exclude` and I do not add 'node_moules' otherwise it is added